project/SparkBuild.scala

```diff
@@ -888,6 +892,22 @@ object PySparkAssembly {
 
 }
 
+object SparkR {
+  import scala.sys.process.Process
+
+  val buildRPackage = taskKey[Unit]("Build the R package")
+  lazy val settings = Seq(
+    buildRPackage := {
+      val command = baseDirectory.value / ".." / "R" / "install-dev.sh"
+      Process(command.toString).!!
+    },
+    (Compile / compile) := {
+      buildRPackage.value
```

Comment:
I think we build SparkR after the compilation of core so the order should probably be switched with the next line.

Reply:
Thanks. I've updated.